src/main/java/bhs/devilbotz/subsystems/Arm.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package bhs.devilbotz.subsystems;
+
+import bhs.devilbotz.Constants;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Arm extends SubsystemBase {
+  /** Creates a new Arm. */
+  private WPI_TalonSRX armMotor;
+
+  private boolean armUp;
+
+  public Arm() {
+    armMotor = new WPI_TalonSRX(Constants.ARM_CAN_ID);
+    armUp = false;
+  }
+
+  public boolean isArmUp() {
+    return armUp;
+  }
```

Comment:
Could we remove this method and boolean? it isnt being used and is not a good way to tell the position.

Reply:
We will be adding limit switches which we can use in conjunction with the encoders to tell the position with a homing sequence